edge-util/src/Microsoft.Azure.Devices.Edge.Util/metrics/IEdgeMetrics.cs

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Util.Metrics
+{
+    public interface IEdgeMetrics
+    {
+        void InitPrometheusMetrics(int port);
```

Comment:
`InitMetrics`? We should probably find a way to remove `Prometheus`

Reply:
Emm, reason I kept Prometheus in the name was because it also starts a http server.. The idea was to have different methods for push/pull modes.. 

Maybe I should rename it to InitMetricsListener..